### ISSUE TITLE
Office.context is an empty object for some emails

### ISSUE THREAD

**Issue by ksobolewskiVL** [OfficeDev/office-js#3026]
For the last few weeks we have been getting reports that for some of the emails the addin in the desktop version of Outlook cannot be run.
No addin works for these emails, not even Script Lab(only the loader appears in it) or Evernote(blank page).
Checking the context I noticed that Office.context returns an empty object.

Addins with this email work correctly through the browser and in the Mac version.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC desktop
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: Version 2211 Build 16.0.15831.20098
* Operating System: Windows
* Browser (if using Office on the web): -
 
## Expected behavior

Outlook Add-ins loads for every email on Windows Desktop Outlook

## Current behavior

Office.context returns {}
I suppose that's why the addins can't load

## Steps to reproduce

1. Open email
2. Open Outlook Addin
3. Open dev tools
4. Check Office.context in console

## Context

It happens randomly, only some of the emails don't work. However, the problem also affects other add-ons.

## Useful logs
I have not been able to reproduce the error and cannot share the email containing the error.

---

What could be the reason for blocking the launch of addins?
Why can office.context return an empty object?

**Comment by timwan10** [comment]
Does the issue happen 100% on the same email? If you switch to a different email (in the reading pane) and go back to the problem email and open an add-in does it still happen? 

Does it happen if the Email is in the Reading Pane or if it is popped out in it's own window? (or both)? 

This is not a known issue. Any more information you can provide on getting a successful repro would greatly help in figuring out the cause.

**Comment by ksobolewskiVL** [comment]
It never loads addins for those emails.

If I open addin for different email, go back to email with problem and open addin there, it won't load.
When I open addin for different email and pin addin, then back to email with problem, addin is still open and loaded, but it looks like it uses previuos email context(or view). Only when I switch to email without problems context(or view) is changed.

It's opened in Reading Pane and won't work for both, opening from email list and double clicked email details window.

I opened Script Lab for email where addin is working, I choosen script to fetch email data from Rest API and received response with email details. Then I pinned addin and I choose email where addin won't work. Then I fetched email data again and I received error message: 
`{"error":{"code":"UnableToReadToken","message":"OAuth token submitted with the request can not be parsed.","innerError":{"oAuthEventOperationId"`
After switch back to valid email I received email data from Rest API.
